### PR TITLE
[9.0][MIG][crm_lead_website] Websites in leads.

### PR DESCRIPTION
**Pull request by yajo**
Now available for v9.

This commit includes all adaptations to v9, and some to new OCA guidelines.

@Tecnativa

### CODE REVIEW

File: crm_lead_website/README.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+feedback.
+
+
+Credits
+=======
+
+Contributors
+------------
+
+* Rafael Blasco <rafaelbn@antiun.com>
```
Comment (pedrobaeza) [inline crm_lead_website/README.rst]: Change emails to new ones

File: crm_lead_website/models/crm_lead.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        return (super(CrmLead, self.with_context(default_website=lead.website))
+                ._lead_create_contact(lead, name, is_company, parent_id))
+
+    @api.multi
+    def on_change_partner_id(self, partner_id):
+        """Recover website from partner if available."""
+        result = super(CrmLead, self).on_change_partner_id(partner_id)
+
+        if result.get("value"):
+            if self.partner_id and self.partner_id.website:
```
Comment (pedrobaeza) [inline crm_lead_website/models/crm_lead.py]: You can't use `self.partner_id`, as old onchange API methods doesn't guarantee that self is the current recordset. You need to make a browse. Once done, you can query directly `partner_id.website`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]

**Review by atchuthan — APPROVED** [review]
Tested in Runbot :+1:

**Review by rafaelbn — APPROVED** [review]
Review comments functionally it's OK

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Waiting for CIs to merge.

**Pull request merged**